velox/functions/lib/Re2Functions.cpp

```diff
@@ -470,6 +470,18 @@ class OptimizedLikeWithMemcmp final : public VectorFunction {
   vector_size_t reducedPatternLength_;
 };
 
+// This functions is constructed when the input is a constant null.
+class AlwaysNull final : public VectorFunction {
```

Comment:
There is AlwaysFailingVectorFunction in VectorFunction.h. If this one doesn't work, let's add a new one in the same place so it can be reused.